feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -231,7 +231,6 @@ class BillingWrapperTest {
 
     @Test
     fun `queryPurchaseHistoryAsync fails if sent invalid type`() {
-        // TODO pull to common once refactoring queryPurchaseHistoryAsync to take RCProductType
```

Comment:
this was leftover from when we had BC4 and BC5 at the same time, and i had split the BillingWrapperTest code up, with some into a "common" file. no longer relevant